wdns/str_to_rdata_ubuf.c

```diff
@@ -131,28 +131,19 @@ str_to_ubuf(const char *str, ubuf *u, const char **retp)
 
 /*
  * str_to_svcparam() translates a "key=val" string to the SVCB/HTTPS wire
- * format as specified in draft-ietf-dnsop-svcb-https-08, section 2.2. Note
- * that some keys don't require a value while others can have more than one.
```

Comment:
why did you remove the comment?  there are a few places here you reviewed what I thought we useful comments.